infrastructures/infrastructure-factory/src/main/java/org/eclipse/che/api/factory/server/scm/kubernetes/KubernetesGitCredentialManager.java

```diff
@@ -167,20 +166,17 @@ public void createOrReplace(PersonalAccessToken personalAccessToken)
   }
 
   /**
-   * Returns username URL segment for git credentials. For OAuth2 tokens it is "oauth2", for others
-   * - {@param personalAccessToken#getScmUserName()} or just "username" string if the token has a
-   * non-null {@param personalAccessToken#getScmOrganization()}. This is needed to support providers
-   * that do not have username in their user object. Such providers have an additional organization
-   * field.
+   * Returns username URL segment for git credentials - {@param
+   * personalAccessToken#getScmUserName()} or just "username" string if the token has a non-null
+   * {@param personalAccessToken#getScmOrganization()}. This is needed to support providers that do
+   * not have username in their user object. Such providers have an additional organization field.
    */
   private String getUsernameSegment(PersonalAccessToken personalAccessToken) {
     // Special characters are not allowed in URL username segment, so we need to escape them.
     PercentEscaper percentEscaper = new PercentEscaper("", false);
-    return personalAccessToken.getScmTokenName().startsWith(OAUTH_2_PREFIX)
```

Comment:
@vinokurig could you clarify why it used to be `oauth2` ? could it have some side-effects?

Reply:
Unfortunately I don't know for sure, this hasn't been changed for a long time. I believe we used it as a stub as we didn't use to have username in the parameters.

Reply:
> could it have some side-effects?

I don't think so because if the stub works (for other providers), username will work in the same way.